runtest.py

```diff
@@ -52,7 +52,7 @@ def loop_suite(self, input_suite_fp, input_max_retry, input_max_run):
                     test_env['ENABLE_PROFILER'] = self.enable_profiler
                     test_env['ENABLE_CHROME_TRACING'] = self.enable_chrome_tracing
                     test_env['DISABLE_AVCONV'] = self.disable_avconv
-                    test_env['CLOSE_BROWSER'] = self.close_browser
+                    test_env['KEEP_BROWSER'] = self.close_browser
```

Comment:
The main() function and Line 132 `run_test_obj = RunTest(enable_profiler_flag, disable_avconv_flag, keep_browser_flag, ...` use the `keep_browser_flag` as input parameter's name, 
so i think rename the `close_browser` of Line 31, 34, 55 will make it more readable, ex `self.keep_browser = keep_browser`.